lib/go-tc/tenants.go

```diff
@@ -19,18 +19,30 @@ package tc
  * under the License.
  */
 
-// GetTenantsResponse is the response for a request for a group of tenants
+// GetTenantsResponse is the response for a request for a group of tenants.
 type GetTenantsResponse struct {
 	Response []Tenant `json:"response"`
 	Alerts
 }
 
-// TenantResponse is the response to a create/update
+// TenantResponse is the response to a create/update.
 type TenantResponse struct {
 	Response Tenant        `json:"response"`
 	Alerts   []TenantAlert `json:"alerts"`
 }
 
+// TenantResponseV40 is the type of a response from Traffic Ops to a PUT, POST,
+// or DELETE request made to its /tenants endpoint in API version 4.0.
+type TenantResponseV40 struct {
```

Comment:
I can't give you a pkg.go.dev link because the name has changed since 5.1.1, but if you look directly above this changed code block you can see a pre-existing `TenantResponse` structure with which this name would collide without `V40`.

Reply:
I see, it's using a `TenantAlert`. I think we can get away with just embedding an `Alerts` in that struct in lieu of `[]TenantAlert`. I can't imagine that would break any real-world client users.

Reply:
That's fine with me, as long as you agree to field the questions from the devs using the client, should there be any

Reply:
I'll tell them to direct their questions to `/dev/null`